business_requirement_crm/models/business.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = "crm.lead"
+
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Project',
+        ondelete='set null',
+    )
+    resource_cost_total = fields.Float(
+        compute='_get_resource_cost_total',
+        string='Total Revenue from BR'
+    )
+
+    @api.one
+    def _get_resource_cost_total(self):
+        linked_brs = self.project_id and self.project_id.br_ids
```

Comment:
Is also a little far for me in memory. But yes I remember that we decided left master_project on the business_requirement. 
Then we might need to modify project to add one2many relation to the `business.requirement` so I'll do as you suggest and modify the br_deliverable.
After that I'll rebase this PR.